README.md

```diff
@@ -96,8 +96,10 @@ A curated list of awesome companies that offer their tools and services for free
 
 - [Crowdin](https://crowdin.com/) `requires-approval` - Localization management platform.
 - [LingoHub](https://lingohub.com/) `requires-approval` - Translation management.
+- [Locale](https://www.localeapp.com/) - Localization and Translation.
 - [POEditor](https://poeditor.com/) `requires-approval` - Localization management platform.
 - [Transifex](https://www.transifex.com/) `requires-approval` - Localization platform for translating digital content.
+- [Weblate](https://weblate.org/) - Web-based continuous localization.
```

Comment:
I took a quick look at Weblate's pricing page and it looks like you have to contact them for Open Source pricing. Is there a review process, or is it based off of something like a GitHub link (where a public repo is automatically included in the Open Source tier)?

If approval is necessary, can we get the `requires-approval` tag added?

```suggestion
- [Weblate](https://weblate.org/) `requires-approval` - Web-based continuous localization.
```

Reply:
Good question, so I'm participating there in several (over ~70) OSS projects, like phpMyAdmin, Debian Handbook, Coala (Py), GNU, Git Manpages. I didn't had to pay for anything (it's free).
I will compared this to the Crowdin, if I'm not mistaken - review process and/or without, depends of maintainer' preferences/settings.
The summary is, yeah, I think we can add the tag:-)